org.firestormviewer.FirestormViewer.json

```diff
@@ -0,0 +1,89 @@
+{
+  "app-id":"org.firestormviewer.FirestormViewer",
+  "branch":"stable",
+  "runtime":"org.freedesktop.Platform",
+  "runtime-version":"19.08",
+  "sdk":"org.freedesktop.Sdk",
+  "command":"firestorm-viewer",
+  "separate-locales":false,
+  "finish-args":[
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--allow=multiarch",
+    "--device=dri",
+    "--filesystem=home",
```

Comment:
Does it require access to entire home?

Reply:
It does run if I use `persist=.firestorm_x64`, but the app doesn't use portals so the user might accidentally save to HOME inside the container, loosing files and the functions of the app that open file explorers don't work.

Reply:
I think `xdg-download` would be preferred here as it can be easily changed by user. Other than that, LGTM.

Reply:
I could add that with xdg-picture and --filesystem=~/.firestorm_64 
I have a dislike of extra hidden home directories, but --persist causes issues on this application.

Reply:
Sounds good! Sorry, I missed your comment somehow.

Reply:
No worries, this submission is in limbo anyway.